### PR TITLE
added return docstring for search

### LINKED ISSUE #3546: Database search() method missing returns documentation
See https://docs.sunpy.org/en/latest/api/sunpy.database.Database.html#sunpy.database.Database.search - the return type of `search()` is currently not documented.

### PR DESCRIPTION
**Pull request by iamrajiv**
### Description

Fixes the documentation of `search()` by adding `Returns` docstring.

Fixes #3546

### CODE REVIEW

File: sunpy/database/database.py [hunk 1]
```diff
@@ -578,6 +578,11 @@ def search(self, *query, **kwargs):
             :class:`sunpy.database.tables.DatabaseEntry` for a list of all
             possible values.
 
+        Returns
+        -------
+        table
```
Comment (dstansby) [inline sunpy/database/database.py]:
```suggestion
        table : list
```

File: sunpy/database/database.py [hunk 2]
```diff
@@ -578,6 +578,11 @@ def search(self, *query, **kwargs):
...
+            List of database entries that satisfy all of the given attributes.
```
Comment (dstansby) [inline sunpy/database/database.py]:
```suggestion
            List of :class:`sunpy.database.tables.DatabaseEntry` that satisfy all of the given attributes.
```

**Comment by iamrajiv** [comment]
@dstansby I have updated the code

**Comment by iamrajiv** [comment]
@dstansby I have updated the code please take a look

File: changelog/3546.doc.rst [hunk 3]
```diff
@@ -0,0 +1 @@
+Fixed documentation of `search()` by adding `Returns` docstring.
```
Comment (Cadair) [inline changelog/3546.doc.rst]:
```suggestion
Fixed documentation of `sunpy.database.Database.search` by adding ``Returns`` docstring.
```

File: sunpy/database/database.py [hunk 4]
```diff
@@ -578,6 +578,12 @@ def search(self, *query, **kwargs):
             :class:`sunpy.database.tables.DatabaseEntry` for a list of all
             possible values.
 
+        Returns
+        -------
+        table : `list`
+            List of :class:`sunpy.database.tables.DatabaseEntry` that
```
Comment (Cadair) [inline sunpy/database/database.py]:
```suggestion
            List of `sunpy.database.tables.DatabaseEntry` that
```

**Comment by Cadair** [comment]
hey @iamrajiv I have added a couple of little syntax tweaks. Also the number in the changelog file should refer to the PR and not the issue :smile:

**Comment by iamrajiv** [comment]
> hey @iamrajiv I have added a couple of little syntax tweaks. Also the number in the changelog file should refer to the PR and not the issue

sorry for inconvenience @Cadair and thank you for guidance. I have implemented your suggestion please take a look

**Pull request merged**